test/accessors/SimulateTxAccessor.spec.ts

```diff
@@ -65,7 +65,7 @@ describe("SimulateTxAccessor", () => {
             const simulation = accessor.interface.decodeFunctionResult("simulate", acccessibleData);
             expect(safe.interface.decodeFunctionResult("getOwners", simulation.returnData)[0]).to.be.deep.eq([user1.address]);
             expect(simulation.success).to.be.true;
-            expect(simulation.estimate).to.be.lte(10000n);
+            expect(simulation.estimate).to.be.lte(14800n);
```

Comment:
I'm a little worried at the changes to the gas usage (it is more than I would have expected). Do we know with code change caused this?

Reply:
Caused due to the optimization, next commit will revert these. Sorry!